src/runtest.rs

```diff
@@ -1524,6 +1524,25 @@ actual:\n\
         }
     }
 
+    // Split library arguments on "-L" to handle paths with spaces properly. Like
+    // split_maybe_args(), empty strings filtered out.
+    fn split_maybe_lib_args(&self, argstr: &Option<String>) -> Vec<String> {
+        if let Some(ref s) = *argstr {
+            s.split("-L")
```

Comment:
What if I have a path like "-L Ooh-La-La"?  Or even if you incorporating spaces in your split, " -L " is also legal in a path.

This will also act weird if people have their own non-`-L` stuff in `target_rustcflags`.